docs/content/guides/6.multistore/3.patterns/2.design/4.different-code-per-store.md

```diff
@@ -1,8 +1,208 @@
 ---
-title: Different markups for individual components
+title: Different Markups for Individual Components
```

Comment:
This changes the title in the navbar. So far other items in the navbar use another casing, it would be good to be consistent. I like your version more, because it uses a standard English heading casing
![CleanShot 2025-03-17 at 13 02 47@2x](https://github.com/user-attachments/assets/ee5b7f62-4953-4ad9-9a89-4d4dff8457e3)

Reply:
Fixed